javascript/javascript3/week3/ownPlaylist.js

```diff
@@ -0,0 +1,56 @@
+
+
+  // add song
+  let myFavorite_songsList=[]; 
+  function addSongs(song_Id,title_,artist_)
+  {
+ 
+    this.songId=song_Id;
+    this.title=title_;
+    this.artist=artist_;
+        let newSong={
+            songId:this.songId,
+            title:this.title,
+            artist:this.artist
+        }
+         myFavorite_songsList.push(newSong);
+
+  }
+
+  let song5=new addSongs(1,"My baby","Soggy socks");
+  let song6=new addSongs(2,"3 nails in wood","The carpenters");
+  
+  function getSongByTitle(title) 
+   {
+     
+    for(let i=0;i<myFavorite_songsList.length;i++)
+    {
+
+        if(myFavorite_songsList[i].title==title)
+        {
+        
+           
+          return  indexNo=myFavorite_songsList.findIndex(x=>x.title===title);// pass anonymous function in findindex function
```

Comment:
if you're using the for loop, `findIndex` is not necessary, you can just `return myFavorite_songList[i]`